core/workspace_svg.js

```diff
@@ -2261,7 +2249,7 @@ Blockly.WorkspaceSvg.prototype.getScale = function() {
  * @package
  */
 Blockly.WorkspaceSvg.prototype.scroll = function(x, y) {
-  Blockly.hideChaff(/* opt_allowToolbox */ true);
+  Blockly.hideChaff(true);
```

Comment:
I would leave the jsdoc comment with the parameter name, personally, just update it to be correct. so that you don't have to go look at the method signature for this to figure out what true means.

Reply:
Done.